translations/pt-br.yaml

```diff
@@ -18,6 +18,7 @@ component:
     edit-translation-text: Editar tradução
   locale-menu:
     select-your-preferred-language: Selecionar idioma
-    locale-en-US: English (US) 
+    locale-en-US: English (US)
     locale-fr-FR: Français
     locale-pt-BR: Português (do Brasil)
+    locale-es: Espanhol
```

Comment:
You brought an interesting point of whether to localize the select options. (I'm not sure yet.)

Reply:
🤔 mmmmm. Let me know